modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -37,6 +37,29 @@ ifdef::ovn[]
 - A protocol that is one of the following protocols: TCP, UDP, and SCTP
 endif::ovn[]
 
+[IMPORTANT]
+====
+You must allow access to the {product-title} API servers or connections to the API endpoints are blocked. For example, you must define egress firewall rules similar to the following:
```

Comment:
I think the context is like this : there is an EgressFirewall rule in the project and the pods in that project want to access API servers. But in the EgressFirewall there is a rule like Deny to 0.0.0.0/0. In this case, the pods in that project access to API server will be blocked in OVN cluster.  To make it work, we has to add allow rule for API servers.